### DISCUSSION TITLE
我想关闭死亡造成的生命值上限下降 How to remove death penalty

### DISCUSSION THREAD

**Discussion by QwIsLosingHair** [Rad586/Forget-Me-Not#76] [Q&A] [answered]
我知道吃新食物可以抵消，但我还是希望不要因为怕死亡而玩游戏畏手畏脚。请问我可以通过删除mod或更改配置关闭生命值上限下降的机制吗？在B站我看到作者您回复过要删除Limited Lives但是我在整合包mod列表中没有发现这个mod。

**Comment by Rad586** [comment, marked as answer]
### **2.10.0 及以上：**
游戏内输入命令 `toggleLimitedLives`。

### **2.10.0 以下：**
**方法一：**搜索 `limited_lives.js` 并用 `function limited_lives(e) {}` 代替其内容；
**方法二：**把游戏难度调成简单或和平。

### **2.10.0 and above:** 
Execute command: `toggleLimitedLives` in game.

### **Before 2.10.0:** 
**Method 1:** Search `limited_lives.js` and replace its content with `function limited_lives(e) {}`.
**Method 2:** Set difficulty to easy or peaceful.
